stylesheet.css

```diff
@@ -487,4 +487,18 @@ body > footer {
   padding-bottom: 1rem;
   line-height: 1.5rem;
   font-weight: 100;
+}
+
+@media (min-width: 992px) {
```

Comment:
Please read about using media queries. For safety (older browsers) is common to use:
`@media only screen and (max-width: 992px)`. And there is a difference between `max-width` and `min-width` - the second one you can use when styling mobile first and going up with width. Also there is difference with using or not using `only` and `screen` words.